test/e2e/e2e.go

```diff
@@ -815,7 +819,17 @@ func deleteMetalLB(metallb *metallbv1beta1.MetalLB) {
 	}, timeout, interval).Should(BeTrue())
 }
 
-var _ = BeforeSuite(func() {
-	_, err := testclient.Client.Namespaces().Get(context.Background(), OperatorNameSpace, metav1.GetOptions{})
-	Expect(err).ToNot(HaveOccurred(), "Should have the MetalLB Operator namespace")
-})
+func getMetalLB() (*metallbv1beta1.MetalLB, error) {
+	metallb := &metallbv1beta1.MetalLB{}
+	if UseMetallbResourcesFromFile {
+		err := loadMetalLBFromFile(metallb, consts.MetalLBCRFile)
+		if err != nil {
+			return nil, err
+		}
+		return metallb, nil
```

Comment:
i think this can be outside the if

Reply:
I would rather not. The metallb is not nil, in case of an error we want to return nil instead of a half populated (possibly no name and namespace) resource .